environment.yml

```diff
@@ -25,6 +25,7 @@ dependencies:
   - esmvalcore 2.10.*
   - fiona
   - fire
+  - fsspec
```

Comment:
this is not needed, it is an indirect dependency via esmvalcore anyway (and I was keen to see where it is imported and used here, but I couldn't see any reference of it?)

Reply:
It's used [here](https://github.com/ESMValGroup/ESMValTool/pull/3227/files#diff-690e995e026e6091df024d4da87c3831ed054c7129b8a1b7f733289481529d3fR32), for TarFileSystem. I'll remove the dependency from the lists.